src/lib/y2network/proposal_settings.rb

```diff
@@ -95,7 +96,26 @@ def create_instance
     def use_network_manager?
       return false unless network_manager_available?
 
-      Yast::Lan.UseNetworkManager
```

Comment:
When the proposal settings are instantiated the package is still not installed so we cannot rely on that method.

https://github.com/yast/yast-network/blob/master/src/modules/Lan.rb#L861